### ISSUE TITLE
False positive `SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse` when using imported function inside switch's default case

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#1688]
```php
<?php

declare(strict_types=1);

namespace X;

use function Foo\yo;

final class X
{
    public function __construct()
    {
        switch (true) {
            default:
                yo();
        }
    }
}
```

With "squizlabs/php_codesniffer": "^3.10.1"

> phpcs: SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse: Type Foo\yo is not used in this file.

**Comment by kukulich** [comment]
The tokenizer was fixed in PHPCS, probably in https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/3.11.0
